packages/utils/src/run-prettier.ts

```diff
@@ -13,6 +12,15 @@ import logger from 'webpack-cli/lib/utils/logger';
 
 export function runPrettier(outputPath: string, source: string): void {
     let prettySource: string = source;
+
+    let prettier;
+    try {
+        // eslint-disable-next-line node/no-extraneous-require
+        prettier = require('prettier');
+    } catch (err) {
+        return fs.writeFileSync(outputPath, source, 'utf8');
```

Comment:
you can also print a warning to let user know that if they install prettier their file can be formatted nicely

Reply:
Done